vendor/templates/vendor/manage/invoice_detail.html

```diff
@@ -27,7 +27,7 @@ <h2>{% trans 'Admin: Order Details' %}</h2>
       <td>$ {{item.total|floatformat:2}}</td>
     </tr>
     {% empty %}
-    <h3> Cart is Empty </h3>
+    <h3> {% trans 'Cart is Empty' %} </h3>
```

Comment:
Remove leading & trailing whitespace in the H3 tag

Reply:
space removed